File: app/src/main/java/me/eagzzycsl/darkroom/ui/FragmentSysApp.java
package me.eagzzycsl.darkroom.ui;

import java.util.ArrayList;

import me.eagzzycsl.darkroom.access.AppList;
import me.eagzzycsl.darkroom.model.OnDeviceApp;


public class FragmentSysApp extends FragmentOnDeviceApp<OnDeviceApp> {
    @Override
    public ArrayList<OnDeviceApp> getOnDeviceApps() {
        return AppList.getSysApps();
    }

}

File: app/src/main/java/me/eagzzycsl/darkroom/ui/FragmentSysApp.kt
package me.eagzzycsl.darkroom.ui

import java.util.ArrayList

import me.eagzzycsl.darkroom.manager.AppList
import me.eagzzycsl.darkroom.model.MetaApp


class FragmentSysApp : FragmentOnDeviceApp() {
    override fun getOnDeviceApps(): ArrayList<MetaApp> {
        return AppList.sysApps
    }
}
